node/src/external/jxcore/jx.h

```diff
@@ -0,0 +1,134 @@
+// Copyright & License details are available under JXCORE_LICENSE file
```

Comment:
I think we should keep the JXCore implementation at least initially. The new API can be plugged in alongside it, that way if there are any issues there we can easily compare.